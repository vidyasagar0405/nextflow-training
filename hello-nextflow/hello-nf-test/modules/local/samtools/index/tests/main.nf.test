nextflow_process {

    name "Test Process SAMTOOLS_INDEX"
    script "../main.nf"
    process "SAMTOOLS_INDEX"

test("reads_son [bam]") {

        when {
            params {
                outdir = "tests/results"
            }
            process {
                """
                input[0] = file("${projectDir}/data/bam/reads_son.bam")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
