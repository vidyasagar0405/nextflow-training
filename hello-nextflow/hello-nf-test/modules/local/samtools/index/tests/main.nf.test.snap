{
    "reads_son [bam]": {
        "content": [
            {
                "0": [
                    [
                        "reads_son.bam:md5,af5956d9388ba017944bef276b71d809",
                        "reads_son.bam.bai:md5,a2ca7b84998218ee77eff14af8eb8ca2"
                    ]
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.3"
        },
        "timestamp": "2025-01-05T16:29:26.666536"
    }
}